charts/fluentd/templates/fluentd-configmap.yaml

```diff
@@ -104,7 +104,7 @@ data:
         {{- if and .Values.global.features.namespacePools.enabled (gt (len .Values.global.features.namespacePools.namespaces.names) 0) }}
         key $.kubernetes.namespace_name
         pattern ^({{ join "|" .Values.global.features.namespacePools.namespaces.names }})$
-        {{- else if .Values.global.manualNamespaceNamesEnabled }}
+        {{- else if or .Values.global.manualNamespaceNamesEnabled .Values.global.disableManageClusterScopedResources }}
```

Comment:
How many different flags do we have for disabling cluster roles and stuff? I wonder if we're overcomplicating this.